fabric/src/main/java/whocraft/tardis_refined/fabric/TardisRefinedFabric.java

```diff
@@ -81,16 +81,17 @@ public void onInitialize() {
         register(SERVER_DATA, new ResourceLocation(TardisRefined.MODID, TardisRefined.MODID + "/" + "hums"), TardisHums.getReloadListener());
 
         if (ModCompatChecker.immersivePortals()) {
-            if (TRConfig.COMMON.COMPATIBILITY_IP.get()) {
-                ImmersivePortals.init();
-                PortalsCompatFabric.init();
-            }
+            //   if (TRConfig.COMMON.COMPATIBILITY_IP.get()) {
+            ImmersivePortals.init();
+            PortalsCompatFabric.init();
+            //     }
```

Comment:
Should this still be commented?

Reply:
Eh, it was more for testing because I kept resetting my runs folder 

An argument could be made for this config to exist, but be on by default. Considering some of our users reading comprehension in the past couple of weeks it would be a good idea